images/ubuntu/toolsets/toolset-2404.json

```diff
@@ -29,7 +29,8 @@
             "arch": "x64",
             "versions": [
                 "18.*",
-                "20.*"
+                "20.*",
+                "22.*"
```

Comment:
1. Lets add new Node to all the supported platforms.
2. Check on the docker images section too for ubuntu-20 and ubuntu-22

Reply:
Sure, I will do.